### ISSUE TITLE
[install] Notice: Undefined property: stdClass::$class

### ISSUE THREAD

**Issue by Quy** [joomla/joomla-cms#19147]
### Steps to reproduce the issue
Install nightly staging.
See notices on the Finalization page.

### Expected result
No notices.

### Actual result
```
Notice: Undefined property: stdClass::$class in C:\xampp\htdocs\joomla-cms-staging\layouts\joomla\form\field\radio.php on line 70
None (Required for basic native multilingual site creation)	

Notice: Undefined property: stdClass::$class in C:\xampp\htdocs\joomla-cms-staging\layouts\joomla\form\field\radio.php on line 70
Blog English (GB) Sample Data
Install Joomla with a few articles and blog related modules like Older Posts, Blog Roll, Most Read Posts.

Notice: Undefined property: stdClass::$class in C:\xampp\htdocs\joomla-cms-staging\layouts\joomla\form\field\radio.php on line 70
Brochure English (GB) Sample Data
Install Joomla with a few pages (a menu with pages Home, About Us, News, Contact Us) and modules like Search, Custom, Login Form.

Notice: Undefined property: stdClass::$class in C:\xampp\htdocs\joomla-cms-staging\layouts\joomla\form\field\radio.php on line 70
Default English (GB) Sample Data	

Notice: Undefined property: stdClass::$class in C:\xampp\htdocs\joomla-cms-staging\layouts\joomla\form\field\radio.php on line 70
Learn Joomla English (GB) Sample Data	

Notice: Undefined property: stdClass::$class in C:\xampp\htdocs\joomla-cms-staging\layouts\joomla\form\field\radio.php on line 70
Test English (GB) Sample Data	
```

**Comment by alikon** [comment]
@Quy can you test #19164 ?

**Comment by ghost** [comment]
closed as having Pull Request #19164
